content/en/news/releases/1.5.x/announcing-1.5/_index.md

```diff
@@ -15,4 +15,97 @@ We are pleased to announce the release of Istio 1.5!
+fewer moving parts which are easier to debug and understand.  For mesh users,
+istiod doesn’t change any of their experience: all APIs and runtime
+characteristics are consistent with the previous components.
+
+Keep your eyes out for a blog post in the coming days devoted to istiod, and the
+benefits of moving to a simpler deployment model.
+
+### A new model for extensibility
+
+Istio has long been the most extensible service mesh, with Mixer plugins
+allowing custom policy and telemetry support and Envoy extensions allowing data
+plane customization. In Istio 1.5 we’re announcing a new model that unifies
+Istio’s extensibility model with Envoy’s, using
+[WebAssembly](https://webassembly.org/) (Wasm). Wasm will give developers the
+ability to safely distribute and execute code in the Envoy proxy -- to integrate
+with telemetry systems, policy systems, control routing and even transform the
+body of a message. It will be more flexible and more efficient, eliminating the
+need for running a Mixer component separately (which also simplifies
+deployments).
+
+Look for blog posts here and from Google, Solo.io and the Envoy community for
+much more detail about this exciting work!
+
+### Easier to use
+
+We’re always making Istio easier to adopt and use, and this release in
+particular has some cool enhancements. Command line installation of Istio using
+[`istioctl`](/docs/reference/commands/istioctl) is now beta for installation and
+will work for most customers in most use cases. Managing your installation via
+an Operator is still alpha, but we continue to improve it with a new
+[IstioOperator API](/docs/reference/config/istio.operator.v1alpha12.pb).
+
+Speaking of `istioctl`, it has over a dozen improvements -- new items it can
+analyze, better validation rules, and better ability to integrate with CI
+systems (look for examples coming soon!). It is
+now an essential tool for understanding the state of a running Istio system and
+for ensuring that config changes are safe. And `istioctl analyze` has graduated
+from the Experimental to the Alpha stage.
+
+### More secure
+
+As always, we are working to make Istio more secure with every release. Some
+highlights with this release:
+[Auto mutual-TLS](docs/tasks/security/authentication/auto-mtls/) has reached the
+Beta stage, and SDS is now stable. We have combined the Node agent and the Pilot
+agent into a single binary, which means we no longer require PodSecurityPolicy.
+We have a new beta version of the Authentication policies as well (see
+`PeerAuthenticationMethod` and `RequestAuthentication` in
+[AuthenticationPolicy](/docs/reference/config/security/istio.authentication.v1alpha1/).
+
+### Better visibility
+
+We continue to invest in making Istio the best way to understand your
+distributed applications. Telemetry v2 now reports metrics for raw TCP
+connections, and we’ve enhanced the support for gRPC workloads by adding
```

Comment:
raw TCP connections in addition to HTTP connections?  cc @mandarjog to correct me

Reply:
Telemetry v2 now reports metrics for plain TCP traffic as well has HTTP traffic that was previously suppprted.

Reply:
I can clarify.